config/overlays/kind-tests/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
nitpick: the formatting/indentation of this yaml isn't consitent with the rest of the YAMLs in this repo (list items are indented 2 ie:
```
foo:
  - bar
```

rather than
```
foo:
- bar
```
)